runtime/src/main/kotlin/generated/_CollectionsNative.kt

```diff
@@ -10,6 +10,8 @@ package kotlin.collections
 // See: https://github.com/JetBrains/kotlin/tree/master/libraries/stdlib
 //
 
+import kotlin.ranges.contains
+import kotlin.ranges.reversed
```

Comment:
Why those imports needed?

Reply:
Otherwise `Iterable` extensions `contains` and `reversed` from `kotlin.collections` are preferred even for ranges using the "same package" priority since we're in `kotlin.collections` package here, see https://youtrack.jetbrains.com/issue/KT-30633 for details